exercicios/para-casa/sistema_de_alunas.py

```diff
@@ -36,24 +36,129 @@ def obter_opcao():
         return codigo_opcao
     
 def incluir_nova_aluna():
-    pass
-    #TODO - Implentar a função
-    
+    print("Por favor insira estes dados: ")
+    nome = input("Nome da aluna: ")
+    sobrenome = input("Sobrenome da aluna: ")
+    turma = input("Turma da aluna: ")
+    lista_presenca = obter_presenca()
+    nota_participacao = float(input("Participação da aluna: "))
+    notas = obter_notas()
+
+    salvar_dados_aluna(nome, sobrenome, turma, lista_presenca, nota_participacao, notas)
+    print("Aluna adicionada com sucesso!")
+
+def obter_presenca():
+    quantidade_aulas = input("Quantidade de aulas: ")
+    aulas = []
+
+    for contador in range(int(quantidade_aulas)):
+        while True:
+            entrada = input(f"Insira a presença da aula #{contador + 1} (True ou False): ")
+            if entrada in ["True", "False"]:
+                presenca = entrada == "True"
+                aulas.append(presenca)
+                break
+            else:
+                print("Entrada inválida. Por favor, insira True ou False.")
+
+    return aulas
+
+def obter_notas():
+    quantidade_notas = input("Quantidade de notas: ")
+    notas = []
+
+    for contador in range(int(quantidade_notas)):
+        while True:
+            entrada = input(f"Insira a nota #{contador + 1}: ")
+            try:
+                nota = float(entrada)
+                if nota >= 0 and nota <= 10:
+                    notas.append(nota)
+                    break
+                else:
+                    print("Nota inválida. Por favor, insira uma nota entre 0 e 10.")
+                    continue
+            except ValueError:
+                print("Entrada inválida. Por favor, insira um número válido.")
+             
+
+    return notas
+
+def salvar_dados_aluna(nome, sobrenome, turma, lista_presenca, nota_participacao, notas):
+    chave = f"{nome} {sobrenome}"
+    dataset[chave] = {
+        "Nome": nome,
+        "Sobrenome": sobrenome,
+        "Turma": turma,
+        "Notas": notas,
+        "Presença": lista_presenca,
+        "Participação": nota_participacao
+    }
+
 def consultar_lista_alunas():
-    pass
-    #TODO - Implentar a função
-    
+    if not dataset:
+        print("Nenhuma aluna cadastrada.")
+        return
+
+    print("\nLista de alunas:")
+    for nome in dataset:
+        print(f"- {nome}")
```

Comment:
Aqui o esperado é trazer o nome e sobrenome da aluna, não a tupla em si. É necessário buscar na relação das chaves do dicionário o valor no índice 0 e 1 da tupla ou dar nomes para cada um deles, desta forma:

**Sugestão:**
```suggestion
    for nome, sobrenome in dataset.keys():
        print(f'Nome: {nome} {sobrenome}')
```